ooniapi/services/oonifindings/src/oonifindings/routers/oonifindings.py

```diff
@@ -0,0 +1,459 @@
+"""
+OONIFindings incidents management
+"""
+
+from datetime import datetime, date, timezone
+from typing import List, Dict
+import logging
+
+from clickhouse_driver import Client as Clickhouse
+from fastapi import APIRouter, Depends, Header, Response, HTTPException
+
+from pydantic import Field
+
+from ..common.routers import BaseModel
+from ..common.dependencies import get_settings, role_required
+from ..common.auth import (
+    hash_email_address,
+    get_account_id_or_raise,
+    get_account_id_or_none, 
+    get_client_role
+)
+from ..common.exceptions import InvalidRequest, OwnershipPermissionError, BaseOONIException
+from ..common.utils import setnocacheresponse, generate_random_intuid
+from ..common.clickhouse_utils import query_click, raw_query, insert_click, optimize_table
+from ..dependencies import get_clickhouse_session
+
+log = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+def utcnow_seconds():
+    return datetime.now(timezone.utc).replace(microsecond=0)
+
+
+class OONIFindingBase(BaseModel):
+    incident_id: str 
```

Comment:
You can use alias to change the serialization: https://docs.pydantic.dev/latest/concepts/fields/#field-aliases